scopes/design/ui/label/index.ts

```diff
@@ -1,5 +1,5 @@
-import * as simulations from './label.simulation';
+import * as compositions from './label.composition';
```

Comment:
I added ThemeCompositions to the compositions of this component because we don't see the styling without it.

Reply:
And why you need to export the compositions of this component?